packages/server/modules/auth/index.ts

```diff
@@ -18,8 +18,7 @@ import {
   getUserByEmail,
   findOrCreateUser,
   validatePasssword,
-  createUser,
-  getUserById
+  createUser
```

Comment:
no IoC refactor in this PR, only deleting getUserById cause its the same as the legacy getUser